### DISCUSSION TITLE
Unable to download 2025 versions

### DISCUSSION THREAD

**Discussion by steinialla** [jeremytammik/RevitLookup#216] [Q&A]
Hi 
I'm unable to download Revit Lookup 2025 both single- and multi user msi.
I always get this "Virus detected" message.
![image](https://github.com/jeremytammik/RevitLookup/assets/4439494/ea182010-d6a2-4871-8f4e-d609b9c49f42)

**Comment by Nice3point** [comment]
It's an interesting topic after that https://github.com/jeremytammik/RevitLookup/issues/207 issue) 

Most likely it's a limitation of your computer's policy, or the antivirus is falsely triggering. RevitLookup is an OpenSource project, all source code is publicly available. 

https://www.virustotal.com/gui/file/2e57d6641c96a5431a9fb49d4267a94194df7afc52c1c7e024f9765251521979/detection
![изображение](https://github.com/jeremytammik/RevitLookup/assets/20504884/60f521e1-2f3b-4fc1-baf4-88049feddc8e)

**Comment by chuongmep** [comment]
If you are using Microsoft Edge, try tune off Windows Defender SmartScreen

**Comment by steinialla** [comment]
Thanks guys for your answers, I managed to download the file after I changed the Protection history in Windows Security Settings. Turning off Windows Defender SmartScreen solely was not sufficient.
